packages/web-components/src/components/video-player/video-player.ts

```diff
@@ -37,7 +37,7 @@ class DDSVideoPlayer extends FocusMixin(LitElement) {
    * Handles `click` event on the video thumbnail.
    */
   private _handleClickOverlay() {
-    this.contentState = VIDEO_PLAYER_CONTENT_STATE.VIDEO;
+    // this.contentState = VIDEO_PLAYER_CONTENT_STATE.VIDEO;
```

Comment:
Thanks @asudoh! Only thing is it looks like you forgot to remove this?

```suggestion
```

Reply:
It was temporary comment-out and I meant to bring it back. Good catch @kennylam!